Task7/Queue.cs

```diff
@@ -0,0 +1,62 @@
+﻿namespace Task7
+{
+    public class Queue<T> : IQueue<T>
+    {
+        private T[] _elements;
+        private int _count;
+
+        public Queue(int capacity)
+        {
+            _elements = new T[capacity];
+            _count = 0;
+        }
+
+        public void Enqueue(T e)
+        {
+            if (_count == _elements.Length)
+            {
+                throw new Exception("Queue is full.");
+            }
+
+            _elements[_count++] = e;
+        }

```

Comment:
not fully effective but ok (try to propose solution without moving elements each time